bitnami/kubectl/1.26/debian-11/Dockerfile

```diff
@@ -23,7 +23,7 @@ ENV HOME="/" \
 COPY prebuildfs /
 SHELL ["/bin/bash", "-o", "errexit", "-o", "nounset", "-o", "pipefail", "-c"]
 # Install required system packages and dependencies
-RUN install_packages ca-certificates curl git jq procps
+RUN install_packages ca-certificates curl git jq procps gettext-base
```

Comment:
Please use `gettext` as in other images like [appsmith](https://github.com/bitnami/containers/blob/main/bitnami/appsmith/1/debian-12/Dockerfile#L26)